modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextStore.java

```diff
@@ -225,6 +231,180 @@ public DurableStoreConnection init(ConfigurationProperties connectionProperties)
     public void deleteBindingInformation(String exchangeName, String boundQueueName) throws AndesException;
 
     /**
+     * Create a new slot in store
```

Comment:
Fixed comments in AndesContextStore with PR https://github.com/AnujaLK/andes/commit/2164a0cfaaba6532fc6ab6f23f774574e9b24b3f

I will keep correcting comment fullstops in code base.